### PR TITLE
[add] registration/login of the user and user store

### PR DESCRIPTION
**Pull request by hamzahdili2001**

### CODE REVIEW

**Comment by hamzahdili2001** [comment]
You can review it without approving it still working on some details

**Comment by MrSakine** [comment]
> You can review it without approving it still working on some details

I'm gonna wait

File: frontend/src/store/userstore.js [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+				this.user.first_name
+			);
+			this.user.last_name = localStorage.setItem(
+				"user.last_name",
+				this.user.last_name
+			);
+		},
+		refreshToken() {
+			axios
+				.post("", {
```
Comment (MrSakine) [inline frontend/src/store/userstore.js]: are you sure the endpoint is correct here ? shouldn't be `api/token/refresh` ?

Comment (hamzahdili2001) [reply frontend/src/store/userstore.js]: this is not working yet

Comment (MrSakine) [reply frontend/src/store/userstore.js]: Alright

**Review by MrSakine — APPROVED** [review]

**Pull request merged**